src/main/java/pokecube/core/database/pokedex/PokedexEntryLoader.java

```diff
@@ -1022,18 +1022,6 @@ else if (vec.length == 3) offsets.add(new double[]
                     }
                 }
 
-                try
```

Comment:
were these removed for a specific purpose?

Reply:
you told me to iirc